tf2onnx/graph_matcher.py

```diff
@@ -102,21 +102,33 @@ def add(self, pattern, op, tensor):
             #       'Name %s is already bound to another pattern' % pattern.name)
             self._name_to_pattern[pattern.name] = pattern
 
-    def _to_pattern(self, pattern_or_name):
+    def _to_pattern(self, pattern_or_name, default=None):
+        """
+        if pattern_or_name cannot match OptypePattern and is not in name_to_pattern, then the function return None
+        """
         if isinstance(pattern_or_name, OpTypePattern):
             return pattern_or_name
 
         if isinstance(pattern_or_name, six.text_type):
-            return self._name_to_pattern[pattern_or_name]
+            get_name_to_pattern = self._name_to_pattern.get(pattern_or_name)
+            if get_name_to_pattern is None:
+                return default
+            return get_name_to_pattern
```

Comment:
directly return self._name_to_pattern.get(pattern_or_name, None)? no need of line 113-115.